website/src/components/Demos/DemoCard/index.tsx

```diff
@@ -29,19 +29,35 @@ const previewProps: IDocumentCardPreviewProps = {
   ],
 }
 
-export default function DemoCard(): JSX.Element {
+export interface DemoCardProps {
+  name: string
+  title: string
+  docLink: string
+  liveDemoLink: string
+}
+
+export default function DemoCard(props: DemoCardProps): JSX.Element {
+  console.log(props)
   return (
-    <DocumentCard aria-label="Demo card" onClickHref="#">
+    <DocumentCard aria-label={props.name}>
       <DocumentCardPreview {...previewProps} />
-      <DocumentCardLocation location={'Scoreboard'} ariaLabel="Scoreboard" />
-      <DocumentCardTitle title={'A scoreboard live demo to show to monitor real time matches'} styles={styles.title} />
+      <DocumentCardLocation location={props.name} locationHref={props.docLink} ariaLabel={props.name} />
+      <DocumentCardTitle title={props.title} styles={styles.title} />
       <DocumentCardTitle title={'Is this recommendation helpful?'} shouldTruncate showAsSecondaryTitle />
       <Separator></Separator>
       <Stack horizontal horizontalAlign="space-between" styles={styles.footer}>
         <StackItem style={styles.footerItem}>
           <Stack horizontal horizontalAlign="space-between">
-            <PrimaryButton text="Try demo" allowDisabledFocus />
-            <Link styles={styles.link}>See details</Link>
+            <PrimaryButton
```

Comment:
Use href property to open the live demo in a new tab? 
<PrimaryButton href="http://bing.com" target="_blank" ...

Reply:
And what if liveDemoLink is not set? hide the button?

Reply:
how about if `liveDemoLink is not set` --> fallback to use `docLink`

the `docLink` always exists